fail2ban/server/failmanager.py

```diff
@@ -43,6 +44,33 @@ def __init__(self):
 		self.__maxRetry = 3
 		self.__maxTime = 600
 		self.__failTotal = 0
+		self.__subnetMask = None
+		
+	def setSubnetMask(self, value):
+		try:
+			self.__lock.acquire()
+			if "." in value: # 255.255.255.0 style mask
+				maskparts = value.split(".")
+				if len(maskparts) != 4:
+					raise ValueError("subnetmask must be valid")
+				for part in maskparts:
+					if int(part) < 0 or int(part) > 255:
```

Comment:
wouldn't `if not (0 <= int(part) <= 255)` be more pythonic and save us mighty second `int(part)` evaluation? ;)